frontend/fingertips-frontend/components/organisms/LineChart/lineChartHelpers.ts

```diff
@@ -176,21 +176,42 @@ export function generateStandardLineChartOptions(
 ): Highcharts.Options {
   const sortedHealthIndicatorData =
     sortHealthDataForAreasByDate(healthIndicatorData);
+  const firstYear = getFirstYearForAreas(sortedHealthIndicatorData);
+  const lastYear = getLatestYearForAreas(sortedHealthIndicatorData);
 
   const sortedBenchMarkData = optionalParams?.benchmarkData
     ? sortHealthDataForAreaByDate(optionalParams?.benchmarkData)
     : undefined;
+  const filteredSortedBenchMarkData = sortedBenchMarkData
+    ? {
+        ...sortedBenchMarkData,
+        healthData:
+          sortedBenchMarkData?.healthData.filter(
+            //shouldLineChartBeShown() in OneIndicatorOneAreaViewPlots ensures these with not be null
+            (data) => data.year >= firstYear! && data.year <= lastYear!
```

Comment:
Yeah we shouldn't be using `!` to enforce non null in production code. If we are confident that these shouldn't be null. We can have an `if` check to ensure these exist and if not throw an error since something has gone very wrong for this to happen